pkg/diff/plan_generator_test.go

```diff
@@ -1,48 +1,60 @@
-package diff_test
+package diff
 
 import (
 	"context"
 	"database/sql"
-	"io"
+	"fmt"
 	"testing"
 
 	_ "github.com/jackc/pgx/v4/stdlib"
+	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/suite"
 	"github.com/stripe/pg-schema-diff/internal/pgengine"
-	"github.com/stripe/pg-schema-diff/pkg/diff"
+	"github.com/stripe/pg-schema-diff/internal/schema"
+	"github.com/stripe/pg-schema-diff/pkg/log"
+	externalschema "github.com/stripe/pg-schema-diff/pkg/schema"
 
 	"github.com/stripe/pg-schema-diff/pkg/tempdb"
 )
 
-type simpleMigratorTestSuite struct {
+type fakeSchemaSource struct {
```

Comment:
Overhauled this test suite a bit with renames. A lot of the naming was vestigial remnants of the old plan function plus an even older implementation of migration planning.